docker-for-aws/upgrade.md

```diff
@@ -36,8 +36,6 @@ Single containers started (for example) with `docker run -d` are
 **not** preserved during an upgrade. This is because they're not Docker Swarm
 objects, but are known only to the individual Docker engines.
 
-_NIB: Please note that if EFS encryption is enabled, upgrade will [recreate the EFS volume](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html){: target="_blank" class="_"}_
```

Comment:
I suggest you should still have something here. As current template breaks things

Reply:
Done!